TF_Service_dotNet/TouchFree/UpdateBehaviour.cs

```diff
@@ -39,6 +51,12 @@ private void OnTimerElapsed(object sender, ElapsedEventArgs e)
                 try
                 {
                     onUpdate?.Invoke();
+                    slowUpdateIteration++;
+                    if (slowUpdateIteration == slowUpdateCount)
```

Comment:
Would it be good practice here to check >= instead of ==, in case of asynchronous execution (IIRC UpdateBehaviour isn't blocking so that could happen). Not really big in this case because a. the value isn't otherwise used and b. is immediately reset but just to avoid any weird concurrency issues, might be nice

Reply:
I made it blocking as part of a recent bug fix so we will only ever have one instance in here at a time, but probably worth doing anyway

Reply:
I've made this change too